gorm/user.go

```diff
@@ -1,10 +1,13 @@
 package gorm
 
 import (
+	"cafapp-returns/apiObjects"
 	"cafapp-returns/jwt"
 	"cafapp-returns/logger"
 	"time"
 
+	"github.com/davecgh/go-spew/spew"
+
 	"github.com/lithammer/shortuuid"
```

Comment:
Unused import. Not saying this is your fault, but these things could be automatically removed if you install the Golang extension for vscode (then as you work, it will ask a bunch of questions if you want to install additional plugins, just click yes to install them all). On a related note, install the eslint plugin as well.